package.json

```diff
@@ -4,24 +4,35 @@
   "description": "A simple style guide generator for Grunt.js",
   "main": "Gruntfile.js",
   "dependencies": {
-    "nunjucks": "3.1.6",
+    "nunjucks": "3.2.0",
     "underscore": "1.8.3",
-    "yaml-front-matter": "3.2.3"
+    "yaml-front-matter": "4.1.0"
   },
   "devDependencies": {
-    "grunt": "1.0.3",
+    "@commitlint/cli": "8.3.4",
+    "@silvermine/eslint-config": "2.3.0",
+    "check-node-version": "4.0.2",
+    "grunt": "1.0.4",
     "grunt-browserify": "5.3.0",
-    "grunt-contrib-jshint": "2.0.0",
-    "grunt-sass": "3.0.2"
+    "grunt-contrib-jshint": "2.1.0",
+    "grunt-sass": "3.1.0",
+    "node-sass": "4.13.0"
   },
   "peerDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-browserify": "^3.8.0",
-    "grunt-sass": "^2.0.0"
+    "grunt": "~1.0.0",
+    "grunt-browserify": "~5.3.0",
+    "grunt-sass": "~3.0.0",
+    "node-sass": "~4.12.0"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "check-node-version --npm 6.13.4",
```

Comment:
FYI: this is what we will do for most (if not all) of our OS repos. We want to continue to run tests against a variety of versions of Node, but ensure that we are always using npm 6.13.4 on each of them.